concent_api/core/message_handlers.py

```diff
@@ -566,7 +566,7 @@ def store_subtask(
         provider        = provider,
         requestor       = requestor,
         state           = state.name,
-        next_deadline   = parse_timestamp_to_utc_datetime(next_deadline),
+        next_deadline   = next_deadline and parse_timestamp_to_utc_datetime(next_deadline),
```

Comment:
In this case it will also fail (because `next_deadline` will be passed to `parse_timestamp_to_utc_datetime`) for ie. `True`, `[0]`etc. Please:
1. Add assert, that if next_deadline is not None, it is int.
2. Change function signature typing for `next_deadline` to be `Union[int, None]`.
3. Change the condition to check explicitly if `isinstance(next_deadline, int) parse_timestamp_to_utc_datetime(next_deadline) else None`

Reply:
as discussed:
ad 1) assert will be changed (in `models` as well)
ad 2) I will use `Optional[int]`
ad 3) will do! ;)

Reply:
done